### PR TITLE
Add some missing nuclides from nuclides.dat

### PR DESCRIPTION
**Pull request by albeanth**
## What is the change?
A simple change to add some nuclides to nuclides.dat to support downstream analysis. 

## Why is the change being made?
It's good to account for all the nuclides that one _might_ care about. 

---

## Checklist

- [x] This PR has only [one purpose or idea](https://terrapower.github.io/armi/developer/tooling.html#one-idea-one-pr).
- [Tests](https://terrapower.github.io/armi/developer/tooling.html#test-it) have been added/updated to verify any new/changed code.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/developer/tooling.html#add-release-notes) have been updated if necessary.
- [x] The [documentation](https://terrapower.github.io/armi/developer/tooling.html#document-it) is still up-to-date in the `doc` folder.
- [x] The dependencies are still up-to-date in `pyproject.toml`.

### CODE REVIEW

**Comment by albeanth** [comment]
@john-science just a heads up. This should be pretty simple to push through. I can handle internal QA-related things. I've pinged @jakehader and Connor for an SME review.

**Comment by john-science** [comment]
@albeanth Can you run the big internal suite of unit tests against this PR? Obviously, functionally, this PR shouldn't make a big difference. But I imagine some "diffs" popping up in your internal integration tests, and who knows maybe someone is checking an exact number in their unit tests somewhere.

**Review by linconnor — CHANGES_REQUESTED** [review]
Two lines don't look right to me. Please double check.
line 2914 now is 70  171 188 0 should be -> 70  118 188 0
line 3092 now is 73  120 195 0 should be -> 73  122 195 0
 
Additionally, it's good to add the source of these added lines, and modify the sentence at the beginning of the file "This file contains a set of 4614 nuclides and their associated metadata"  4614 -> 4624

File: armi/resources/nuclides.dat [hunk 1]
```diff
@@ -2911,6 +2911,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 70  115 185 0   YB    1.84967901031e+02 0.00000000000e+00 1.60000000000e-07 0.000000
 70  116 186 0   YB    1.85960897004e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 70  117 187 0   YB    1.86964769503e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+70  171 188 0   YB    1.87966850000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (albeanth) [inline armi/resources/nuclides.dat:2914]:
```suggestion
70  118 188 0   YB    1.87966850000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```

File: armi/resources/nuclides.dat [hunk 2]
```diff
@@ -3087,6 +3089,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 73  119 192 0   TA    1.91975147454e+02 0.00000000000e+00 2.20000000000e+00 0.000000
 73  120 193 0   TA    1.92977401897e+02 0.00000000000e+00 inf               0.000000
 73  121 194 0   TA    1.93981277391e+02 0.00000000000e+00 inf               0.000000
+73  120 195 0   TA    1.94972540000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (albeanth) [inline armi/resources/nuclides.dat:3092]:
```suggestion
73  122 195 0   TA    1.94972540000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```

**Review by linconnor — APPROVED** [review]
The added nuclides data look good to me.

**Review by jakehader — APPROVED** [review]

**Comment by john-science** [comment]
I can verify but not validate this change.

Holler when you guys want to merge this.

File: armi/resources/nuclides.dat [hunk 3]
```diff
@@ -2911,6 +2911,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 70  115 185 0   YB    1.84967901031e+02 0.00000000000e+00 1.60000000000e-07 0.000000
 70  116 186 0   YB    1.85960897004e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 70  117 187 0   YB    1.86964769503e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+70  118 188 0   YB    1.87966850000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:2914]: Checked

File: armi/resources/nuclides.dat [hunk 4]
```diff
@@ -2979,6 +2980,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 71  117 188 0   LU    1.87973172133e+02 0.00000000000e+00 inf               0.000000
 71  118 189 0   LU    1.88970840395e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 71  119 190 0   LU    1.89969390037e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+71  120 191 0   LU    1.90971560000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:2983]: Checked

File: armi/resources/nuclides.dat [hunk 5]
```diff
@@ -3087,6 +3089,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 73  119 192 0   TA    1.91975147454e+02 0.00000000000e+00 2.20000000000e+00 0.000000
 73  120 193 0   TA    1.92977401897e+02 0.00000000000e+00 inf               0.000000
 73  121 194 0   TA    1.93981277391e+02 0.00000000000e+00 inf               0.000000
+73  122 195 0   TA    1.94972540000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:3092]: Checked

File: armi/resources/nuclides.dat [hunk 6]
```diff
@@ -4246,6 +4249,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 95  156 251 0   AM    2.51082286436e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 95  157 252 0   AM    2.52084870103e+02 0.00000000000e+00 inf               0.000000
 95  158 253 0   AM    2.53086880851e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+96  134 230 0   CM    2.30046089000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:4252]: Checked

File: armi/resources/nuclides.dat [hunk 7]
```diff
@@ -4272,6 +4276,8 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 96  157 253 0   CM    2.53086880851e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 96  158 254 0   CM    2.54090599607e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 96  159 255 0   CM    2.55091048349e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+97  134 231 0   BK    2.31045560000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:4279]: Checked

File: armi/resources/nuclides.dat [hunk 8]
```diff
@@ -4377,6 +4383,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 100 161 261 0   FM    2.61092087368e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 100 162 262 0   FM    2.62095806125e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 100 163 263 0   FM    2.63096254866e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+101 142 243 0   MD    2.43070833000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:4386]: Checked

File: armi/resources/nuclides.dat [hunk 9]
```diff
@@ -4429,6 +4436,7 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 102 163 265 0   NO    2.65094690616e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 102 164 266 0   NO    2.66098409372e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 102 165 267 0   NO    2.67098858114e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+103 145 248 0   LR    2.48074127000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:4439]: Checked

File: armi/resources/nuclides.dat [hunk 10]
```diff
@@ -4477,6 +4485,8 @@ Z	N	A	S	El	  Mass, amu	        Abundance, frac.  Half-life (sec)   Spontaneous F
 104 165 269 0   RF    2.69097293874e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 104 166 270 0   RF    2.70101012631e+02 0.00000000000e+00 1.11110000000e+01 0.000000
 104 167 271 0   RF    2.71101461372e+02 0.00000000000e+00 1.11110000000e+01 0.000000
+105 147 252 0   DB    2.52076730000e+02 0.00000000000e+00 1.11110000000e+01 0.000000
```
Comment (jakehader) [inline armi/resources/nuclides.dat:4488]: Checked

**Comment by jakehader** [comment]
> I can verify but not validate this change.
> 
> Holler when you guys want to merge this.

I checked them @john-science.

**Comment by john-science** [comment]
@jakehader @albeanth This appears complete to me.  Status?

**Pull request merged**